components/pages/user/StudyAccess.tsx

```diff
@@ -63,22 +120,19 @@ const StudyAccess = (): ReactElement | null => {
 			</h2>
 
 			<p>You are authorized to submit data for the following studies:</p>
-			{userCanSubmitDataForAllStudy ? (
-				<b>All Studies</b>
-			) : (
-				<ul
-					css={css`
-						${theme.typography.subheading};
-						font-weight: normal;
-						color: ${theme.colors.accent_dark};
-						margin-bottom: 1rem;
-					`}
-				>
-					{effectiveScopes?.map((scope) => (
-						<li key={scope}>{scope}</li>
-					))}
-				</ul>
-			)}
+			<StudiesSection
+				title="Clinical studies"
+				isAdmin={!!userIsClinicalAdmin}
+				hasAccess={!!userHasClinicalAccess}
+				scopes={effectiveClinicalScopes}
+			/>
+
+			<StudiesSection
+				title="Environmental studies"
+				isAdmin={!!userIsEnvironmentalAdmin}
+				hasAccess={!!userHasEnvironmentalAccess}
+				scopes={effectiveEnvironmentalScopes}
+			/>
```

Comment:
this got me thinking whether showing items you have no access for is good UX, but strategically, could lead to people asking how to submit data in the other side they're not part of "yet" 👍